qa/os/src/test/java/org/elasticsearch/packaging/test/EnrollmentProcessTests.java

```diff
@@ -68,6 +71,40 @@ public void test10AutoFormCluster() throws Exception {
         assertThat(makeRequest("https://localhost:9200/_cluster/health"), containsString("\"number_of_nodes\":2"));
     }
 
+    public void test20DockerAutoFormCluster() throws Exception {
+        assumeTrue("only docker", distribution.isDocker());
+        // First node
+        installation = runContainer(distribution(), builder().envVar("ELASTIC_PASSWORD", "password"));
+        verifyContainerInstallation(installation);
+        verifySecurityAutoConfigured(installation);
+        waitForElasticsearch(installation);
+        assertThat(makeRequestAsElastic("https://localhost:9200/_cluster/health", "password"), containsString("\"number_of_nodes\":1"));
+        final String node1ContainerId = Docker.getContainerId();
+        Shell.Result createTokenResult = installation.executables().createEnrollmentToken.run("-s node");
+        assertThat(Strings.isNullOrEmpty(createTokenResult.stdout), is(false));
+        final List<String> noWarningOutputLines = Arrays.stream(createTokenResult.stdout.split("\\n"))
```

Comment:
```suggestion
        String createTokenResult = installation.executables().createEnrollmentToken.run("-s node").stdout;
        assertThat(createTokenResult, not(emptyOrNullString());
        final List<String> noWarningOutputLines = createTokenResult.lines()
```
Note that `.lines()` would need to be replaces with `Arrays.stream()` in `7.16`.

Reply:
Thanks, this is 8.0+ only so I won't backport this to 7.x